examples/worlds/flat_seabed.sdf

```diff
@@ -0,0 +1,161 @@
+<?xml version="1.0" ?>
+<!--
```

Comment:
Knit:  provide a description of this file like: https://github.com/gazebosim/gz-sim/blob/dc3fb1124fa58439abaec56f1301bb70a3ec7852/examples/worlds/buoyancy.sdf#L2-L19

```suggestion
<!--
This example shows how to use the Doppler Velocity Log (DVL) sensor in gazebo.
To see the dvl output simply run this example and in a second terminal run:
   gz topic --echo -t /dvl/velocity
You should see the output in a seperate thread.
```

Also remove MBARI copyright on this file since its your contribution.